docs/usage/deep-linking.md

```diff
@@ -25,7 +26,7 @@ Otherwise, Swagger-UI generates an implicit operationId by combining the operati
 
 > I'm using Swagger-UI in an application that needs control of the URL fragment. How do I disable deep-linking?
 
-This functionality is disabled by default, but you can pass `deepLinking: false` into Swagger-UI as a configuration item to be sure.
+This functionality is disabled by default, but you can pass `deeplinking: false` into Swagger-UI as a configuration item to be sure.
```

Comment:
Maybe we should maintain the case of the name `deepLinking`

Reply:
Change needed: retain the casing 😄